components/Button/index.tsx

```diff
@@ -6,11 +6,12 @@ export type ButtonType = {
   theme?: 'default' | 'blue';
   loading?: boolean;
   children: React.ReactNode;
+  width?: 'default' | 'full-width';
```

Comment:
케밥케이스를 사용한 이유가 있을까요?

```suggestion
  full?: boolean;
```
해서 사용하는게 더 보기 좋지 않을까요?

Reply:
네 바꿨습니다.